packages/terra-signature/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-signature",
+  "main": "lib/Signature.js",
+  "private": true,
+  "version": "0.1.0",
```

Comment:
This needs to be 0.0.0.  When we release the version gets bumped from the "current version" to whatever we decide (in this case it'll be 0.1.0).